### PR TITLE
feat(groups): added topbar menu item for pending invitations

### LINKED ISSUE #5781: Make group invitations more visible
Current version - Elgg 1.8.15

New users find it counter-intuitive to look for Group Invitations via the Groups list page.  It would be nice to have Invitations display on the User's profile and/or Dashboard as well.

### PR DESCRIPTION
**Pull request by jdalsem**
fixes #5781

### CODE REVIEW

File: mod/groups/start.php [hunk 1]
```diff
@@ -862,6 +865,54 @@ function groups_members_menu_setup($hook, $type, $menu, $params) {
...
+	if (empty($count)) {
+		return;
+	}
+	
+	$result = $hook->getValue();
+	
+	// Invitations
+	$text = elgg_echo('groups:invitations');
+	$title = $text;
+	if ($count) {
```
Comment (jeabakker) [inline mod/groups/start.php]: this a always true because of line 888

File: mod/groups/start.php [hunk 2]
```diff
@@ -862,6 +865,54 @@ function groups_members_menu_setup($hook, $type, $menu, $params) {
...
+		$title = elgg_echo('groups:invitations:pending', [$count]);
+	}
+	
+	$result[] = \ElggMenuItem::factory([
+		'name' => 'groups:user:invites',
+		'text' => $text,
+		'badge' => $count ?: null,
```
Comment (jeabakker) [inline mod/groups/start.php]: same

File: mod/groups/views/default/groups/invitationrequests.php [hunk 3]
```diff
@@ -14,8 +14,8 @@
 	$vars['offset'] = get_input('offset', 0);
 	$vars['count'] = groups_get_invited_groups($user->guid, false, ['count' => true]);
 	$invitations = groups_get_invited_groups($user->guid, false, [
-		'limit' => $limit,
-		'offset' => $offset
+		'limit' => $vars['limit'],
```
Comment (jeabakker) [inline mod/groups/views/default/groups/invitationrequests.php]: elgg_extract

**Review by jeabakker — DISMISSED** [review]

**Comment by hypeJunction** [comment]
Why not add tests? It doesn't take much effort and everyone has peace of mind. 10 years later the likes of me won't be cursing you for adding a ton of untested code.

**Comment by jdalsem** [comment]
> Why not add tests?

uh... for registering a menu item??

**Comment by hypeJunction** [comment]
> uh... for registering a menu item??

Yes, because to test it, you will have to generate invitations, so you will also test the invitation API along the way

**Comment by jdalsem** [comment]
added #11494 for the tests

**Pull request merged**